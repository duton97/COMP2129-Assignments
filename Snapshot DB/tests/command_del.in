SET a 1 2 3
DEL a
SET b 1 2 3
DEL b
SET c 1 2 3
DEL c
BYE
