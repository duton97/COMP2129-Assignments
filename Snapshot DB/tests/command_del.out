> ok

> ok

> ok

> ok

> ok

> ok

> bye
